draft-ietf-tls-tls13.md

```diff
@@ -5010,11 +5048,12 @@ pseudorandom function (PRF).  In addition, as long as this function is truly
 one way, it is not possible to compute traffic keys from prior to a key change
 (forward secrecy).
 
-TLS does not provide security for data which is sent after a traffic secret is
-compromised (backward secrecy). Compromise of a traffic secret also compromises
-future confidentiality and integrity.  Systems which want backward secrecy need
-to do a fresh handshake and establish a new session key with an (EC)DHE
-exchange.
+TLS does not provide security for data which is communicated on a connection
+after a traffic secret of that connection is compromised i.e., post-compromise
```

Comment:
Nit: I would s/compromised i.e., post-/compromised. That is, TLS does not provide post-/